tooltester/src/main/java/io/dockstore/tooltester/runWorkflow/WorkflowRunner.java

```diff
@@ -478,17 +478,23 @@ private void addSingleMetricToQa(String metricName) {
                 .build();
         GetMetricStatisticsResponse response = cloudWatchClient.getMetricStatistics(request);
         Double sumOfDataPoints = 0D;
-        Double maxDataPoint = 0D;
+        Double maxDataPoint = response.datapoints().get(0).average();
+        Double minDataPoint = response.datapoints().get(0).average();
+        final int numberOfDataPoints = response.datapoints().size();
         for (Datapoint datapoint: response.datapoints()) {
             sumOfDataPoints += datapoint.average();
             maxDataPoint = max(maxDataPoint, datapoint.average());
+            minDataPoint = min(minDataPoint, datapoint.average());
             // datapoint.average() is not actually obtaining the average. This is because the ECS container is only
             // collecting metrics every minute, and we have asked for a minute by minute metric breakdown.
             // So, we are getting the only statistic collected for each minute.
         }
-        Double averageOfDataPoints = sumOfDataPoints / response.datapoints().size();
+        final Double averageOfDataPoints = sumOfDataPoints / numberOfDataPoints;
+
         runMetrics.putAdditionalPropertiesItem(metricName + "_AVERAGE", averageOfDataPoints);
         runMetrics.putAdditionalPropertiesItem(metricName + "_MAX", maxDataPoint);
+        runMetrics.putAdditionalPropertiesItem(metricName + "_MIN", minDataPoint);
+        runMetrics.putAdditionalPropertiesItem(metricName + "_NUMBER_OF_DATAPOINTS", numberOfDataPoints);
 
```

Comment:
Lines 481 to 497 implement the functionality requested in this [comment](https://ucsc-cgl.atlassian.net/browse/SEAB-5179?focusedCommentId=44836).


@kathy-t